openwisp_controller/pki/static/pki/js/show_org_field.js

```diff
@@ -0,0 +1,16 @@
+django.jQuery(function ($) {
+    'use strict';
+    var operationType = $('.field-operation_type select');
+    // function for operation_type switcher
+    var showFields = function () {
+        var orgField = $('.form-row.field-organization'),
+            value = operationType.val();
+        if (value === 'import') {
+            orgField.show();
+        }
+    };
+    showFields();
+    operationType.on('change', function (e) {
+        showFields();
+    });
+});
```

Comment:
you can simplify this to:

```javascript
django.jQuery(function ($) {
    'use strict';
    operationType.on('change', $('.form-row.field-organization').show);
});
```

The organization field always needs to be shown, but it's ok to hide it initially to make it clear that the user needs to select one of the initial choices (create new or import existing).